### DISCUSSION TITLE
Theme tabler lang file path?

### DISCUSSION THREAD

**Discussion by backpack-operations** [Laravel-Backpack/community-forum#804] [Q&A (Help)] [answered]
Hello team,
I just received this on email:
"Hi,

I’m using backpack pro on 3 projects.
For one of them I went the theme tabler route.

Everything works fine, but I’m stuck on theme-tabler translations :

This key for example :
backpack.theme-tabler::theme-tabler.color-mode-description

Whatever I do I can’t manage to get Laravel to apply my translations.

I tried to put my file in the normal path :
/lang/vendor/backpack/theme-tabler/fr/theme-tabler.php

But translations are not applied.
Then I tried every other possible path but no luck.
I even tried to put it in /resources/lang like in old versions of laravel, instead of /lang, but no way.

I « solved » it by overwriting the switch_theme.blade.php view, but it’s just not the way it should be.

Any idea ?"

**Comment by pxpm** [comment, marked as answer]
Hey @backpack-operations hope you are doing well. 

I've just updated the docs on how to translate theme strings.

Please let me know if they are not clear enough or if there is any issue with them: https://backpackforlaravel.com/docs/6.x/base-themes#translate-theme-strings-1

Cheers
